src/coredump.rs

```diff
@@ -429,12 +429,12 @@ mod elfcore {
     }
 }
 
+#[cfg(target_arch = "x86_64")]
 #[cfg(test)]
 mod test {
     use super::*;
     use py_spy_testdata::get_coredump_path;
 
-    #[cfg(target_pointer_width = "64")]
```

Comment:
Why was this inner cfg insufficient?

Reply:
ARM64 will also match this `cfg`, but it seems that the coredump file is generated from x86 thus cause unaligned access on ARM64.